client/pom.xml

```diff
@@ -61,5 +61,15 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.swagger</groupId>
+            <artifactId>swagger-annotations</artifactId>
+            <version>1.5.22</version>
```

Comment:
Why is swagger-annotations declared separately with a different version?

Reply:
Not sure this was intentional. Feel free to open a PR!